plaso/parsers/firefox_cache.py

```diff
@@ -194,6 +198,10 @@ def _GetFirefoxConfig(self, file_object, display_name):
       file_object (dfvfs.FileIO): a file-like object.
       display_name (str): display name.
 
+    Returns:
+      firefox__cache_config: namedtuple containing the block size and first
```

Comment:
`firefox__cache_config` => `firefox_cache_config` (remove the double underscore)